server/package-lock.json

```diff
@@ -9,7 +9,7 @@
       "version": "1.0.0",
       "license": "ISC",
       "dependencies": {
-        "bcryptjs": "^2.4.3",
```

Comment:
I noticed that both server lock files have been modified, although I didn't see a reason why this is necessary for this PR. Please check

Reply:
> I noticed that both server lock files have been modified, although I didn't see a reason why this is necessary for this PR. Please check

When I used "npm i" it got added , we have used this library for password hashing